serverless/experimental/wasm/main.go

```diff
@@ -126,7 +126,7 @@ func integrate() js.Func {
 		}
 
 		ctx := context.Background()
-		newCp, err := log.Integrate(ctx, *cp, logStorage, rfc6962.DefaultHasher)
+		newCp, err := log.Integrate(ctx, *&cp.Size, logStorage, rfc6962.DefaultHasher)
```

Comment:
Does just `cp.Size` not work?

Reply:
Hah, good spot. I think that must have been VS code suggesting an auto complete and I really didn't notice it adding the ampersand. It must have assumed the pointer dereference was super important.